### PR TITLE
Update lcobucci/jwt to 5.0.0

### LINKED ISSUE #32643: lcobucci - jwt
As a developer I want to have framework up2date so : 
- I have a secure plateform
- New features 
- more performant prlateform in some case

## Description 

Upgrade [lcobucci - jwt](https://packagist.org/packages/lcobucci/jwt) to,  at least, 5.50 version.

See also : 
[frameworks - PHP dependancies](https://www.notion.so/prestashopcorp/Les-cons-quence-en-fonction-des-versions-de-PHP-support-es-5fa9b7e6654a495cbcbbf86cb82fee91)
and 
[dependencies](https://miro.com/app/board/uXjVMMFa4_Y=/?share_link_id=447161604159)

## AT

PS should work (ISO) with the 5.50 version of  [lcobucci - jwt](https://packagist.org/packages/lcobucci/jwt)

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update lcobucci/jwt to 5.0.0
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is green and https://github.com/M0rgan01/ga.tests.ui.pr/actions/runs/5132603960 is green
| Fixed ticket?     | Fixes #32643 
| Related PRs       | -
| Sponsor company   | -

### CODE REVIEW

**Comment by M0rgan01** [comment]
Tests: https://github.com/M0rgan01/ga.tests.ui.pr/actions/runs/5132603960

File: composer.json [hunk 1]
```diff
@@ -252,12 +252,6 @@
             "Tests\\": "tests/"
         }
     },
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/PrestaShop/jwt"
-        }
-    ],
```
Comment (M0rgan01) [inline composer.json:260]: It may be interesting to archive the repo

**Review by boherm — DISMISSED** [review]

**Review by PululuK — DISMISSED** [review]

**Review by FabienPapet — DISMISSED** [review]

File: composer.json (hunk 1 above)
Comment (matks) [reply composer.json:260]: As soon as this is merged, please ping a GitHub administrators to do it ;) like me, @jolelievre, @kpodemski ...

**Review by matthieu-rolland — COMMENTED** [review]
There's one failing test in your autom tests...  seems like I can't restart it myself

**Comment by boherm** [comment]
@matthieu-rolland 
> There's one failing test in your autom tests... seems like I can't restart it myself

Yeah, this test seems to failed systematically also on my PRs.

**Comment by M0rgan01** [comment]
Rebase after lock conflicts @FabienPapet @PululuK @boherm

**Review by boherm — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by nicosomb** [comment]
Thank you @M0rgan01 !

**Pull request merged**